gregBorchers/README.md

```diff
@@ -0,0 +1,153 @@
+### Homework #4
```

Comment:
Strong work throughtout, and great console.logs! There's one comment further on, but that's it.